apps/expo/package.json

```diff
@@ -49,7 +49,7 @@
   "dependencies": {
     "@bacons/link-assets": "^1.1.0",
     "@capsizecss/core": "^3.0.0",
-    "@coinbase/wallet-mobile-sdk": "^0.3.1",
+    "@coinbase/wallet-mobile-sdk": "0.3.1",
```

Comment:
iOS pod install is failing on eas, so trying this. I am not sure if this will work though!